src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/ReportController.kt

```diff
@@ -69,6 +70,20 @@ class ReportController(context: ActionContext,
         return RunReportViewModel(context, metadata, gitBranches)
     }
 
+    fun getListReports(): List<Map<String, Any?>>
```

Comment:
Can we pin down this Any? It might be null if no report versions, but if there are, can it just be a Date or a date string? Don't need the whole report version object.

Reply:
Have created a data object (`DateWithReport`) for this - please just let me know if that makes sense.